### PR TITLE
Update Prototype Kit docs support page

### PR DESCRIPTION
**Pull request by kellylee-gds**

### CODE REVIEW

File: docs/v13/views/support.html [hunk 1]
```diff
@@ -22,25 +22,14 @@
 
      <h1 class="govuk-heading-xl">Support</h1>
 
-   <p>If you've got a question, idea or suggestion, get in touch with the GOV.UK Prototype team.</p>
+	<p>The GOV.UK Prototype Kit is currently maintained by the [GOV.UK Design System team](https://design-system.service.gov.uk/get-in-touch/). There is only [minimal support in place](https://github.com/alphagov/govuk-prototype-kit/issues/2389) as the current version is stable.</p>
```
Comment (36degrees) [inline docs/v13/views/support.html]: I need to double check but I'm pretty sure markdown links won't work here – they'll need to be HTML links instead (like the Slack link in the next paragraph).

**Comment by 36degrees** [comment]
I've updated and [force pushed](https://github.com/alphagov/govuk-prototype-kit-docs/compare/ed5f6a8499665abb606eaf814e07095db98da758..883961e49467be253df37827d541ac256a50ddb7) a change to:

- use HTML links rather than markdown links (which don't work in this context)
- tweak the wording to align with what we're using elsewhere

**Review by stevenjmesser — APPROVED** [review]
It's possibly worth linking to or duplicating the content written elsewhere about the levels of maintenance and support we're able to offer given the constraints. 'Maintenance' or 'support' can sometimes be read as full-on feature development but the other docs are quite explicit at how deeply we're able to work on it given other commitments. 

I'll approve this but the above is a suggestion you could take in before merging.

**Comment by 36degrees** [comment]
I've [force-pushed](https://github.com/alphagov/govuk-prototype-kit-docs/compare/4e9450f01868e91535192a7ff892868318c9d5e9..883961e49467be253df37827d541ac256a50ddb7) to remove https://github.com/alphagov/govuk-prototype-kit-docs/commit/4e9450f01868e91535192a7ff892868318c9d5e9 as we need to use HTML links in the Prototype Kit docs

**Review by 36degrees — APPROVED** [review]

**Pull request merged**